tests/typegen/types.gen.ts

```diff
@@ -10,206 +10,220 @@ declare global {
 export interface NexusGenInputs {
   CreatePostInput: {
     // input type
-    author: string // ID!
-    geo: Array<Array<number | null>> // [[Float]!]!
-    name: string // String!
-  }
+    author: string; // ID!
+    geo: Array<Array<number | null>>; // [[Float]!]!
+    name: string; // String!
+  };
```

Comment:
Would be great if you could prettify that file so that there's no unnecessary diffs

Reply:
Was going to say the same, thought that should already happen pre-commit.

Reply:
Oh, need to change https://github.com/graphql-nexus/schema/blob/2de6f8971e5caac24af416342708c05218edc142/tests/typegenPrinter.spec.ts#L17-L18 to ```prettierConfig: path.join(__dirname, '../.prettierrc'),```